src/framework/server.ts

```diff
@@ -128,6 +130,24 @@ function setupExpress(
       return {
         ...opts,
         context: settingsGiven.context(req),
+        customFormatErrorFn: error => {
+          const colorlessMessage = stripAnsi(error.message)
+
+          if (process.env.NEXUS_STAGE === 'dev') {
+            resolverLogger.error(error.stack ?? error.message)
```

Comment:
I'm printing the error as the `event` here to keep the colors. Didn't find a better solution. I'm not sure how bad it is to have different logs between dev and prod

Reply:
Not bad. This looks like a use-case for #261.